### ISSUE TITLE
Time and dates in download history always wrong

### ISSUE THREAD

**Issue by BitGrub** [Nexus-Mods/web-issues#1344]
## Bug/error being reported:
The time and dates listed in my download history are inaccurate. I regularly catch myself downloading mods hours "before" they're even published or updated. This makes it extremely difficult to keep track of mods that have updates available.

## Steps to reproduce:
1. Download a mod that was very recently uploaded from somewhere in a different time zone (probably).
2. Check https://www.nexusmods.com/[GameTitleGoesHere]/users/myaccount?tab=download+history for inconsistencies.

**Comment by mrmason** [comment]
Hi, are you able to provide your user id please ? 

Also, what timezone are you in ?

**Comment by mrmason** [comment]
tracking this internally as `nexusmods/-/issues/749`

**Comment by BitGrub** [comment]
I'm currently in BST (GMT+1) but in the site settings I was set to +2 (I tend to travel a lot) but I think the timestamps should be consistent regardless, no?

My profile link is https://www.nexusmods.com/users/17880434

**Comment by mrmason** [comment]
Yes, they should be - there are a number of different layers that this page goes through to generate the date you see - there is a lot of heavy lifting involved!

One of them seems to be using what we assume is the server time rather than the user's time - we've raised a bug internally and will investigate.

Thanks for the report.
